packages/tentales/src/index.js

```diff
@@ -0,0 +1,134 @@
+const Koa = require("koa")
+const bodyParser = require("koa-bodyparser")
+const fetch = require("node-fetch")
+const logger = require("./utils/logger")
+const middlewareRender = require("./middlewares/render")
+
+const MIDDLEWARE_ORDER = [
+  "first",
+  "render",
+  "service-renderer",
+  "service-editor",
+  "service-data",
+  "last"
+]
+
+module.exports = function tenTales(config) {
+  const server = new Koa()
+  const services = {}
+
+  const serviceMiddlewares = Object.keys(config.services)
+    .map(serviceName => {
+      /**
+       * Logger
+       */
+      const log = logger.createServiceLogger(`${serviceName} service`)
+      const ttLog = logger.createServiceLogger(`tt -> ${serviceName}`)
+
+      /**
+       * Access services
+       */
+      const serviceConfig = config.services[serviceName]
+      services[serviceName] = async ({ type, payload }) => {
+        const host =
```

Comment:
Is this really needed, it could be a nice API to just say `self` but compared to `http://localhost:1234` it seams a bit silly?

Reply:
We need _some way_ (not saying that this is a good dito) to determine if a service is an external service or an internal (which then needs to be started on this machine). Search the code for "self", there are a few other places where this check is needed.